SpatialGDK/Extras/core-sdk.version

```diff
@@ -1 +1 @@
-13.3.1
+13.4.0
```

Comment:
Do our projects need an update with this?

Reply:
Yes they do, will create PRs for them